test/base/AppWithDefaultMockFixtures.scala

```diff
@@ -81,6 +81,7 @@ trait AppWithDefaultMockFixtures extends BeforeAndAfterEach with GuiceOneAppPerS
   protected val fakeTransportEquipmentNavigator: TransportEquipmentNavigator           = new FakeTransportEquipmentNavigator(onwardRoute)
   protected val fakeDepartureTransportMeansNavigator: DepartureTransportMeansNavigator = new FakeDepartureTransportMeansNavigator(onwardRoute)
   protected val fakeConsignmentItemNavigator: ConsignmentItemNavigator                 = new FakeConsignmentItemNavigator(onwardRoute)
+  protected val fakeItemNavigator: AdditionalReferenceNavigator                        = new FakeItemNavigator(onwardRoute)
```

Comment:
val can be renamed to `fakeAdditionalReferenceNavigator`

Reply:
Done